### PR TITLE
Added the option to remove low-tier upgrades

### PR DESCRIPTION
**Pull request by Znuff**
This will run in the main loop (is there a better way?).

### CODE REVIEW

**Comment by GreatOldOne** [comment]
How about instead of just changing the element to none, change the buy box to gold and set the cost to MAX and make it unclickable.

**Comment by Znuff** [comment]
I personally like the free space: http://i.imgur.com/fDke975.png

**Comment by NotTsunami** [comment]
This could actually complement #124 really well if both are merged. That way then it would only go for the higher tier if this is check (Which would require another commit, but I believe it would be a really good idea, and is wouldplease a few others who are iffy on #124).

File: autoPlay.user.js [hunk 1]
```diff
@@ -252,6 +254,29 @@ function disableParticles() {
 	}
 }
 
+function doRemoveLowTier() {
+  var map = {
```
Comment (xPaw) [inline autoPlay.user.js:258]: Can we have a global "enum" array of upgrades? Would make way more sense than these ids

**Comment by xPaw** [comment]
Having an option to remove these upgrades is good, but you should also handle re-showing them if player decides to unhide them without requiring a page refresh.

**Comment by rossengeorgiev** [comment]
You shouldn't hide the lower tier upgrades. Eventually they become more cost effective.

**Comment by iakat** [comment]
Closed, see #150.

**Pull request closed without merging**